VideoWeb/VideoWeb/ClientApp/src/app/on-the-day/models/base-self-test.component.ts

```diff
@@ -89,4 +90,10 @@ export abstract class BaseSelfTestComponentDirective implements OnInit {
             this.logger.debug(testcallScore.toJSON());
         }
     }
+
+    async skipSelfTest() {
+        console.log('user skipped the self test!');
+        const reason = new DisconnectedCall('Conference terminated by another participant');
```

Comment:
Could we change the reason to something like "Self test skipped"

Reply:
updated